### ISSUE TITLE
Unable to reproduce cat results after retraining

### ISSUE THREAD

**Issue by alan-smith-vt** [AntonMu/TrainYourOwnYOLO#112]
So my main question boils down to this: Was the cat example trained on the 100 images in the training folder? If so, why can I not train a new model using these images and obtain the results in the example
?

I have been able to run the code fine including the gpu portion, but the accuracy of the results is really really low for me. After training with a simple example of detecting rocks in a video game I was able to obtain some bounding boxes, though most were missing or wrong. In a lot of cases I would simply have nothing detected at all in the test images.

I figured I would take a step back and try using the 100 cat pictures from the example data. This resulted in the same issue. Not a single test image had a box detected in it.

During training of the cat images it ran for 80/102 Epochs before stopping with a loss of 14.7 and val_loss of 40.1. I feel like I must be missing something obvious here. Shouldn't it be showing at least some results? (Note again I was able to get some bounding boxes with a trivial example dataset, so it's not an issue of broken testing. It's just lower accuracy than seems rational).

![image](https://user-images.githubusercontent.com/48192097/78162677-3628cc00-7415-11ea-8f3a-e8e4c2dd7303.png)

![image](https://user-images.githubusercontent.com/48192097/78162724-4640ab80-7415-11ea-999e-f976cd20625d.png)

![image](https://user-images.githubusercontent.com/48192097/78162748-535d9a80-7415-11ea-81d3-3bd0cbd4f429.png)
